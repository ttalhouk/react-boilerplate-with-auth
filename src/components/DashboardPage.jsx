import React from 'react';

const DashboardPage = () => (
  <div>
    Main components here
  </div>
)

export default DashboardPage;
