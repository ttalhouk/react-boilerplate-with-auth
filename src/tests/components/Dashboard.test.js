import React from 'react';
import {shallow} from 'enzyme';

import DashboardPage from '../../components/DashboardPage.jsx';

describe('<DashboardPage />',() => {
  it('should render the dashboard page correctly', () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
  });

})
