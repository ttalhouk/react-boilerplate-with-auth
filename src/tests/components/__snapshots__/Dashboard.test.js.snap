// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardPage /> should render the dashboard page correctly 1`] = `
<div>
  Main components here
</div>
`;
